modules/db.rb

```diff
@@ -0,0 +1,21 @@
+module DB
+  YAML_FILE_PATH = 'library_db.yaml'.freeze
+
+  class << self
+    def store(entity)
+      File.open(YAML_FILE_PATH, 'a') { |file| file.write entity.to_yaml }
```

Comment:
do you override all the data in that file and put the `entity`?
if it is - you better not to do it